### PR TITLE
[RFR] Remove

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,27 +216,55 @@ Creates a query to update one entity.
...
 - data: a literal specifying the field to update
 
-### deleteOne
+### remove
 
 ```js
-import deleteOne  from 'co-postgres-queries/queries/deleteOne';
-deleteOne({ table, idFields, returnFields })(db)(ids);
+import remove  from 'co-postgres-queries/queries/remove';
+remove({ table, idField, returnFields })(db)(ids);
```
Comment (djhi) [inline README.md]: `remove` does not support multiple id fields like `removeOne` ?

File: README.md [hunk 2]
```diff
@@ -216,27 +216,54 @@ Creates a query to update one entity.
...
+        field: 'value'
+    }
+    ```
+    will update only row for which field equal 'value'
+
+### removeOne
+
+```js
+import removeOne  from 'co-postgres-queries/queries/removeOne';
+removeOne({ table, idField, returnFields })(db)(ids);
```
Comment (djhi) [inline README.md]: s/idField/primaryKey/

File: README.md [hunk 3]
```diff
@@ -216,27 +216,54 @@ Creates a query to update one entity.
...
 
 - table: the table name
-- idFields: list of key fields used to select the entity (default: `id`)
+- primaryKey: one or more fields representing the primary key. Accept array or single value. (default: `id`)
 - returnFields: list of fields retrieved by the query
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`
+- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`. All configured idField must be given a value.
```
Comment (djhi) [inline README.md]: s/idField/ids/

File: README.md [hunk 4]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`. All configured idField must be given a value.
+- identifier: either a single value for a single primaryKey field, or a literal if several fields:`{ id1: value, id2: otherValue }`. All configured primaryKey fields must be given a value.
```
Comment (djhi) [inline README.md]: shouldn't it be named `primaryKey` like in the others ?

File: README.md [hunk 5]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
...
 - data: a literal specifying the field to update
 
-### deleteOne
+### remove
 
 ```js
-import deleteOne  from 'co-postgres-queries/queries/deleteOne';
-deleteOne({ table, idFields, returnFields })(db)(ids);
+import remove  from 'co-postgres-queries/queries/remove';
+remove({ table, filterFields, returnFields })(ids);
```
Comment (djhi) [inline README.md]: if it does not use id fields anymore, shouldn't `ids` be renamed as `filters` ?

File: README.md [hunk 6]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
...
+removeOne({ table, primaryKey, returnFields })(identitfier);
 ```
 
 Creates a query to delete one entity.
 
 #### Configuration
 
 - table: the table name
-- idFields: list of key fields used to select the entity (default: `id`)
+- primaryKey: one or more fields representing the primary key. Accept array or single value. (default: `id`)
```
Comment (djhi) [inline README.md]: s/Accept array or/Accept either an array or a/

File: README.md [hunk 7]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
...
 - returnFields: list of fields retrieved by the query
 
 #### Parameters
 
-- ids: either a single value for a single id, or a literal for several id:`{ id1: value, id2: otherValue }`
+- identitfier: either a single value for a single primaryKey field, or a literal if several fields:`{ id1: value, id2: otherValue }`. All configured primaryKey fields must be given a value.
```
Comment (djhi) [inline README.md]: Shouldn't this be renamed as `primaryKey` and accept either an array or a value ?

File: README.md [hunk 8]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
...
+```
 
 Allow to create a query to delete several entity at once
 
 #### Configuration
 
 - table: the table name
 - fields: list of fields to insert
-- identifier: the field used to select the entity to delete
+- primaryKey: One or more fields representing the primary key. Accept array or single value. (default: `id`)
```
Comment (djhi) [inline README.md]: s/Accept array or/Accept either an array or a/

File: README.md [hunk 9]
```diff
@@ -249,41 +249,73 @@ Creates a query to update one entity.
...
 
 #### Parameters
 
-- ids: list of ids of the entity to delete
+- identifierList: list of identifier either an array of single value for a single primaryKey field, or an array of literal if several fields:`[{ id1: value, id2: otherValue }, ...]`. All configured primaryKey fields must be given a value.
```
Comment (djhi) [inline README.md]: Shouldn't this be renamed as primaryKey and accept either an array or a value ?

File: README.md [hunk 10]
```diff
@@ -103,7 +103,7 @@ An array of literal objects in the form of:
 
 ```js
 import selectOne  from 'co-postgres-queries/queries/selectOne';
-selectOne({ table, idFields, returnFields })(db)(entity)
+selectOne({ table, idFields, returnFields })(entity)
```
Comment (djhi) [inline README.md]: Shouldn't `idFields` be renamed as `primaryKey` and accept either an array or a value ?

Comment (ThieryMichel) [inline README.md]: done in the next PR

File: README.md (hunk 4 above)
Comment (ThieryMichel) [inline README.md]: nope, it is the values that will be passed to the primaryKey fields, aka the identifier of a document.

File: README.md (hunk 5 above)
Comment (ThieryMichel) [inline README.md]: I'll do it in the following PR. (I fear the rebase)

File: README.md (hunk 6 above)
Comment (ThieryMichel) [inline README.md]: same

File: README.md (hunk 7 above)
Comment (ThieryMichel) [inline README.md]: nope, it is correct as is

File: README.md (hunk 8 above)
Comment (ThieryMichel) [inline README.md]: next PR

File: README.md (hunk 9 above)
Comment (ThieryMichel) [inline README.md]: nope

**Pull request merged**